selectrum.el

```diff
@@ -376,6 +384,20 @@ Used to display STRING according to DOCSIG-FUNC from metadata."
      (format "%s" docsig)
      'face 'selectrum-completion-docsig)))
 
+(defun selectrum--remove-default-from-prompt (prompt)
+  "Remove the indication of the default value from PROMPT.
+Selectrum has its own methods of indicating the default value, making
+other methods redundant."
+  (let ((regexps selectrum--minibuffer-default-in-prompt-regexps))
```

Comment:
This should probably all be wrapped in `save-match-data`, just because it's surprising when utility functions change the match data unexpectedly :)